### ISSUE TITLE
Errors with problem 87 solutions

### ISSUE THREAD

**Issue by serk3ftin** [oxalorg/4ever-clojure#66]
In [problem 87](https://4clojure.oxal.org/#/problem/87):
```
(= (__ 3 4 5 60) '(= (+ (* 3 4) 5) 60))
```
should be:
```
(= (__ 3 4 5 60) '(= (* 3 4 5) 60))
```
and the last two:
```
(= (__ 1 2 10 100 2001) '(= (+ 1 (* 2 10 100)) 2001)
(= (__ 1 2 10 100 1300) '(= (* (+ 1 2 10) 100) 1300)
```
are missing an ending parenthesis

**Comment by seebs** [comment]
yeah, i just ran into these issues also. no wonder there's no solutions in the solutions archive.

**Comment by oxalorg** [comment]
@serk3ftin or @seebs do y'all have a working solution for this problem I can test with?

**Comment by borkdude** [comment]
@oxalorg Isn't that what PR #74 is for?

**Comment by oxalorg** [comment]
Sorry I meant a working code solution for this problem so I can test the #74 PR! I guess I can spend some time going through the problem and coming up with a function :)

**Comment by borkdude** [comment]
Ah I see... The solutions archive doesn't have a solution for this one?

**Comment by seanstrom** [comment]
Here's a link to [my solution for problem 87](https://gist.github.com/seanstrom/f0d1119c24f4d3cb48de33b6ea59425c), hope this helps 😸
